pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -402,6 +402,7 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				continue
 			}
 
+			m.Lock()
```

Comment:
nit: I think this would read slightly easier to follow if we called `m.Lock()` twice, each time before invoking the methods accessing `containerMap`. Just easier to track where and why the lock is needed.

Reply:
I'm actually happy with the way it is currently (I've been stress testing it like this in a soak environment over the last 2 weeks without issue). If you feel strongly though, I can add small helper functions to do the locking around each call.